pcap4j-core/src/main/java/org/pcap4j/core/PcapHandle.java

```diff
@@ -197,7 +197,7 @@ public void setDlt(DataLinkType dlt) throws PcapNativeException, NotOpenExceptio
   /**
    *
    * @return a fraction part of a timestamp of a packet captured in a current thread.
-   *         The value represents the number of microseconds.
+   *         The value represents the number of microseconds or nanoseconds.
```

Comment:
This is, you know, confusing. I will mark this and getTimestampInts as deprecated and add a new api to get timestamp in more refined way.